.github/workflows/publish.yml

```
@@ -0,0 +1,24 @@
+name: Publish
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build-n-publish:
+    name: Build and Publish to PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+      - name: Build
+        run: >-
+          python setup.py bdist
```

For simplicity, I'd suggest just one-lining it:

```suggestion
        run: python setup.py bdist
```